spec/models/bathroom_spec.rb

```diff
@@ -6,4 +6,20 @@
   it { should validate_presence_of(:street_address) }
   it { should validate_presence_of(:zip_code) }
   it { should validate_presence_of(:description) }
+
+  describe '.search' do
+    context 'there are results' do
+      it 'should search zip codes' do
+        expect(Bathroom.search("02116")).to eq Bathroom.where(
+        "zip_code = ?",
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.